build/room_ensurer/room_ensurer.py

```diff
@@ -227,9 +230,26 @@ def _get_room(self, server_name: str, room_alias_prefix: str) -> Optional[RoomIn
         )
         return RoomInfo(room_id=room_id, aliases=existing_room_aliases, server_name=server_name)
 
+    def _create_server_user_power_levels(self) -> Dict[str, Any]:
+
+        server_admin_power_levels: Dict[str, Dict[str, int]] = {"users": {}}
+        for server_name in self._known_servers:
+            username = f"admin-{server_name}".replace(":", "-")
+            user_id = f"@{username}:{server_name}"
+            server_admin_power_levels["users"][user_id] = 50
```

Comment:
if possible don't use "magic" numbers here and in 242 -- what are these power levels? there should be some constants for this in `synapse`, no?

Reply:
AFAIK Im not aware of any constants given. In Matrix itself it is a loose concept to give power levels from 0 to 100. In the documentation they declare 50 to be a "moderator" (whatever that means) and 100 as an administrator,. But you can give powerlevels from 0 to 100. Most of them default to either [0,50,100]

Reply:
thanks for the explanation!